OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -41,7 +41,13 @@ public HarvesterDockSequence(Actor self, Actor refinery, int dockAngle, bool isD
 			DragOffset = dragOffset;
 			DragLength = dragLength;
 			Harv = self.Trait<Harvester>();
-			StartDrag = self.CenterPosition;
+
+			/*
```

Comment:
The best place for the comment would be before you assign to `startDrag` in Tick.